app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error.get("line_number", 0),
+        "column": error.get("column_number", 0),
+        "message": error.get("text", ""),
+        "name": error.get("code", ""),
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
```

Comment:
You could use `map` here as well, e.g.:
```suggestion
        "errors": list(map(format_linter_error, errors)),
```